template/src/interfaces.ts

```diff
@@ -1,21 +1,21 @@
 import { Stream } from 'xstream';
 import { DOMSource, VNode } from '@cycle/dom';
+import { StateSource, Reducer } from '@cycle/state';
```

Comment:
Do you need to importe it here if you use `export from` after that?

Reply:
Not for the export, but for the interface below